<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnProductos">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
border-radius:20px;
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(165, 157, 157);</string>
    </property>
    <property name="text">
     <string>Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCliente">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border-radius:20px;
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: #0F3966</string>
    </property>
    <property name="text">
     <string>Registro de Clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVenta">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border-radius:20px;
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(56, 56, 56);</string>
    </property>
    <property name="text">
     <string>Punto de Venta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border-radius:20px;
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: #0F3966</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/cct/WhatsApp Image 2022-11-13 at 11.17.32 PM (1).jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/el gordo en blanco.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>-10</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/cct/WhatsApp Image 2022-11-14 at 2.35.20 AM (2).jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/menu.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEmpleado">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border-radius:20px;
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: #0F3966</string>
    </property>
    <property name="text">
     <string>Registro de Empleados</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>btnProductos</zorder>
   <zorder>btnCliente</zorder>
   <zorder>btnVenta</zorder>
   <zorder>btnSalir</zorder>
   <zorder>label</zorder>
   <zorder>btnEmpleado</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Logo13.qrc"/>
 </resources>
 <connections/>
</ui>
